python/ray/_raylet.pyx

```diff
@@ -898,16 +898,17 @@ cdef class CoreWorker:
 
         return RayObjectsToDataMetadataPairs(results)
 
-    def object_exists(self, ObjectRef object_ref):
+    def object_exists(self, ObjectRef object_ref, memory_store_only=False):
```

Comment:
Is the name of this kwarg clear, or would something like `not_in_plasma` be clearer?